figures/helpers.py

```diff
@@ -188,6 +194,15 @@ def prev_day(val):
     return days_from(val, -1)
 
 
+def utc_yesterday():
```

Comment:
Would you mind adding a test case for this function? One happy scenario should be enough.

Reply:
will do